src/components/AttachmentModal.tsx

```diff
@@ -519,7 +519,7 @@ function AttachmentModal({
                                 setDownloadButtonVisibility={setDownloadButtonVisibility}
                             />
                         ) : (
-                            !!sourceForAttachmentView &&
+                            (!!sourceForAttachmentView || Object.hasOwn(transaction?.pendingFields ?? {}, 'waypoints')) &&
```

Comment:
This is new. Why are we adding this?

Reply:
Now because we don't pass the image for distance transaction with pending waypoints the Attachement view will not display as `!!sourceForAttachmentView` will be falsy 
https://github.com/Expensify/App/blob/10ff3ee875e69595849f52856c317cd3fe408224/src/libs/ReceiptUtils.ts#L26-L28